expo/gbe/views/edit_act_techinfo_view.py

```diff
@@ -77,20 +78,30 @@ def EditActTechInfoView(request, act_id):
             except:
                 pass
             re_set.insert(0, existing_rehearsal)
+            re_set = sorted(re_set,
```

Comment:
I fixed the insert --> append, but I disagree on the loop.

The for loop is iterating through shows, there is 1 rehearsal set (re_set) per show, and each set should be sorted separately.  It only needs sorting if the act has an existing rehearsal choice (that's when the existing sorted order gets disrupted). 

Reviewing it, though, I found a case where sort should execute less frequently, and there's less screwing with the list, for the same outcome.  Now it will only re-sort the list when the rehearsal that the person is signed up for had to be added to the list - that happens when the rehearsal was fully booked.